### PR TITLE
Adds using power monitor and detecting computer suspend.

### PR DESCRIPTION
**Pull request by damencho**
Depends on https://github.com/jitsi/jitsi-meet-electron-utils/pull/48, new jitsi-meet-electron-utils release and bumping the package.json version.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const windowStateKeeper = require('electron-window-state');
 const {
     initPopupsConfigurationMain,
     getPopupTarget,
-    setupAlwaysOnTopMain
```
Comment (saghul) [inline main.js]: Is this no longer necessary?

Comment (damencho) [reply main.js]: That is a mistake, will fix it.

Comment (damencho) [reply main.js]: Fixed.

File: app/features/conference/components/Conference.js [hunk 2]
```diff
@@ -265,10 +266,13 @@ class Conference extends Component<Props, State> {
         new RemoteControl(iframe); // eslint-disable-line no-new
         setupAlwaysOnTopRender(this._api);
         setupWiFiStats(iframe);
+        setupPowerMonitorRender(iframe);
 
+        this._api.on('suspendDetected', (event: Event) => {
```
Comment (hristoterezov) [inline app/features/conference/components/Conference.js]: Maybe add `this. _onVideoConferenceEnded` as a value for the listener? You may need to do:`this. _onVideoConferenceEnded = this._onVideoConferenceEnded.bind(this)` in the constructor but I'm not sure.

**Pull request merged**